gst/nnstreamer/include/nnstreamer_plugin_api_if.h

```diff
@@ -0,0 +1,43 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * GStreamer/NNStreamer Tensor-IF
+ * Copyright (C) 2020 Gichan Jang <gichan2.jang@samsung.com>
+ */
+/**
+ * @file	nnstreamer_plugin_api_if.h
+ * @date	28 Oct 2020
+ * @brief	NNStreamer APIs for tensor_if custom condition
+ * @see		https://github.com/nnstreamer/nnstreamer
+ * @author	Gichan Jang <gichan2.jang@samsung.com>
+ * @bug		No known bugs except for NYI items
+ *
+ */
+
+#ifndef __NNS_PLUGIN_API_IF_H__
+#define __NNS_PLUGIN_API_IF_H__
+
+#include <stdint.h>
+#include "tensor_typedef.h"
```

Comment:
Revise header includes: include all headers that are used in this .h file and do not include headers not used in this .h file.
There is no one using stdint and you didn't include glib headers while using glib types. (and so on for GST types)

Reply:
Update header files. Thanks.